### PR TITLE
Specialized chart - Ternary Scatter and filled

### PR DESCRIPTION
**Pull request by priyatharsan**
closes: https://github.com/plotly/plotly.github.io/issues/427

### CODE REVIEW

File: _posts/scientific/2016-07-11-how-to-make-a-filled-ternary-plot.md [hunk 1]
```diff
@@ -1,53 +1,63 @@
...
+    - copy: "After adding data, go to the 'Traces' section under the 'Structure' menu on the left-hand side. Choose the 'Type' of trace, then choose 'Ternary Scatter' under 'Specialized' chart type."
+      img: "![Choose chart](../static/images/ternary-scatter-plot/ternary-scatter-type.png)"
+    - copy: "Note that each data point on a ternary plot is plotted according to its relative composition with respects to the 3 main axes. For example, the row (10, 0, 90) represents a data point that is made up of 10% of axis A, 0% of axis B and 90% of axis C, this sums up to 100% of this data point's composition with relation to the three axes."
+      img: "![Data Format](../static/images/ternary-filled-plot/ternary-filled-data-format.png)"
+    - copy: "In addition, each column represents an axis for the data points. To plot all your data points on the ternary graph, you will need to map the data columns on the grid to the axes by specifying the values for the attributes 'A', 'B' and 'C' from their respective dropdown menus. Once this mapping is complete, the data points you've entered into the grid will appear on the ternary plot."
```
Comment (cldougl) [inline _posts/scientific/2016-07-11-how-to-make-a-filled-ternary-plot.md]: @priyatharsan can you add more information and discuss more about the fill aspect of the filled ternary

File: _posts/scientific/2016-07-11-how-to-make-a-filled-ternary-plot.md [hunk 2]
```diff
@@ -1,53 +1,63 @@
...
-      img: "![Plot Data](../static/images/How-to-make-a-filled-ternary-plot/data-plotting.png)"
-    - copy: "To add another trace (layer), create 3 more columns and populate them with your data for the next trace. You can also reuse the same columns, if relevant, on multiple traces. Once your data for the second trace is added, you will have to map your columns to axes for the second trace, just like we did for the first trace. To do so, click on the '+Trace' button in the upper right corner of the GRAPH panel. Assign the columns you've created for the second trace to their respective A, B, C axes. You should now see two traces on your filled ternary plot."
-    - copy: "Tip: if you have multiple traces to add, the 'Collapse All' button may be useful to easily get back to the specific trace that you want to work on."
-      img: "![Add Trace](../static/images/How-to-make-a-filled-ternary-plot/second-trace.png)"
+    - copy: "After adding data, go to the 'Traces' section under the 'Structure' menu on the left-hand side. Choose the 'Type' of trace, then choose 'Ternary Scatter' under 'Specialized' chart type."
```
Comment (cldougl) [inline _posts/scientific/2016-07-11-how-to-make-a-filled-ternary-plot.md]: I'd suggest mentioning here that though we're using the ternary scatter trace the fill will be added subsequently

Comment (priyatharsan) [reply _posts/scientific/2016-07-11-how-to-make-a-filled-ternary-plot.md]: done 👍

File: _posts/scientific/2016-07-11-how-to-make-a-filled-ternary-plot.md (hunk 1 above)
Comment (priyatharsan) [reply _posts/scientific/2016-07-11-how-to-make-a-filled-ternary-plot.md]: I have added the info about the filled area in the Style section.

**Comment by cldougl** [comment]
💃

**Pull request merged**